base/fft/FFTW.jl

```diff
@@ -101,16 +101,16 @@ alignment_of(A::FakeArray) = Int32(0)
 # FFTW's api/import-wisdom-from-file.c file].
 
 function export_wisdom(fname::AbstractString)
-    f = ccall(:fopen, Ptr{Void}, (Cstring,Ptr{UInt8}), fname, "w")
+    f = ccall(:fopen, Ptr{Void}, (Cstring,Cstring), fname, "w")
```

Comment:
Right, though if you have the time/will to add a short comment explaining this, it wouldn't hurt, and will avoid repeating this next time somebody bumps on it.